internal/internal_task_handlers.go

```diff
@@ -188,6 +188,14 @@ type (
 	historyMismatchError struct {
 		message string
 	}
+
+	processedTask struct {
```

Comment:
If someone has a better name then this I welcome any suggestions

Reply:
How about `preparedTask` since it's not yet processed?

Reply:
I called it processed because the SDK had done some processing on it, like extracting SDK flags and reordering events. Words like prerpare or preprocess is probably better